imdb_django/management/commands/import_names.py

```diff
@@ -0,0 +1,68 @@
+# myapp/management/commands/import_name_data.py
+
+import csv
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from imdb_django.models import Name, Title
+from helper import log_info
+
+class NameDataImportCommand(BaseCommand):
+    help = "Load data from TSV file into Name model"
+
+    def add_arguments(self, parser):
+        parser.add_argument("tsv_file", type=str, help="Path to the TSV file")
+
+    def handle(self, *args, **options):
+        tsv_file_path = options["tsv_file"]
+
+        with open(tsv_file_path, "r", encoding="utf-8") as tsv_file:
+            tsv_reader = csv.DictReader(tsv_file, delimiter="\t")
+
+            name_data_to_create = []
+
+            with transaction.atomic():
+                row_count = 0
+
+                for row in tsv_reader:
+                    name_data_to_create.append(self.process_row(row, row_count))
+                    row_count += 1
+
+                # Use bulk_create to insert the rows with ignore_conflicts=True
+                Name.objects.bulk_create(name_data_to_create, ignore_conflicts=True)
+
+        # Log a message to indicate completion
+        log_info(f"\nData import completed. Loaded {row_count} rows.")
+
+    def process_row(self, row, row_count):
+        n_const = row["nconst"]
+        primaryName = row["primaryName"]
+        birthYear = int(row["birthYear"]) if row["birthYear"] != "\\N" else None
+        deathYear = int(row["deathYear"]) if row["deathYear"] != "\\N" else None
+        primary_Profession = row["primaryProfession"]
+        known_for_titles_str = row["knownForTitles"]
```

Comment:
Please update variable names in lines 38 to 42, names should be as per pep8 standards